src/applications/claims-status/tests/e2e/01.claim-status.v2.cypress.spec.js

```diff
@@ -0,0 +1,82 @@
+import TrackClaimsPageV2 from './page-objects/TrackClaimsPageV2';
+import claimsList from './fixtures/mocks/lighthouse/claims-list.json';
+import claimDetails from './fixtures/mocks/lighthouse/claim-detail-closed.json';
+import claimDetailsOpen from './fixtures/mocks/lighthouse/claim-detail.json';
+
+describe('A user can navigate to their claim details of a closed claim', () => {
+  it('Shows the correct status for the claim - C30820', () => {
+    const trackClaimsPage = new TrackClaimsPageV2();
+    trackClaimsPage.loadPage(claimsList, claimDetails);
+    trackClaimsPage.verifyInProgressClaim(false);
+    cy.axeCheck();
+  });
+});
+
+describe('A user can navigate to their claim details of an open claim', () => {
+  it('Shows the correct status for the claim - C30820', () => {
+    const trackClaimsPage = new TrackClaimsPageV2();
+    trackClaimsPage.loadPage(claimsList, claimDetailsOpen);
+    trackClaimsPage.verifyInProgressClaim(true);
+    cy.axeCheck();
+  });
+});
+
+describe('A user with more than 3 contentions can click "Show full list" button and see more contentions', () => {
+  it('Has "Show full list" button which reveals more contentions when clicked', () => {
+    const trackClaimsPage = new TrackClaimsPageV2();
+    trackClaimsPage.loadPage(claimsList, claimDetails);
+    trackClaimsPage.verifyInProgressClaim(false);
+    trackClaimsPage.verifyContentions();
+    cy.axeCheck();
+  });
+});
+
+describe('A user that has tracked items can view recent activity', () => {
+  it('Shows recent activity for a user with tracked items', () => {
```

Comment:
should add this is for a closed claim and id add a test for an open claim as well